src/Workspaces/Core/Portable/Formatting/Engine/AbstractFormatEngine.OperationApplier.cs

```diff
@@ -172,7 +172,17 @@ private bool ApplyForceLinesOperation(AdjustNewLinesOperation operation, int pai
             {
                 var triviaInfo = _tokenStream.GetTriviaData(pairIndex);
 
-                var indentation = _context.GetBaseIndentation(_tokenStream.GetToken(pairIndex + 1));
+                var indentation = 0;
+                switch (_context.OptionSet.GetOption(FormattingOptions.SmartIndent, _tokenStream.FirstTokenInStream.Token.Language))
+                {
+                    case FormattingOptions.IndentStyle.None:
```

Comment:
I'm not sure you've captured the intent of the bug here.  I think that if the indentation is set to block or smart, the formatter should simply not touch it, even if it fixes other whitespace/adjusts casing, etc of the rest of the line.